README.md

```diff
@@ -13,7 +13,7 @@ The Data Structures Game is a competitive online game. The goal of this game is
 1. [Install Anaconda](https://docs.anaconda.com/anaconda/install/)
 2. Create a minimal-empty Python environment:
 - ```conda create --name dsg python=3.8.5```
-- ```source activate dsg```. Note that ```dsg``` is just a name. You may choose differently. The rest of the documentation assumes you named the environment ```dsg```.
+- ```source activate dsg``` (or ```conda activate dsg```). Note that ```dsg``` is just a name. You may choose differently. The rest of the documentation assumes you named the environment ```dsg```.
```

Comment:
I had a pull request up to make this change to the README but saw your PR doing the same so I added this in here. 

Just added "conda activate dsg" since as of conda 4.4.0 this is the preferred method to activate the virtual environment